core/src/main/java/dk/alexandra/fresco/framework/sce/evaluator/BatchedProtocolEvaluatorPerformanceDecorator.java

```diff
@@ -0,0 +1,31 @@
+package dk.alexandra.fresco.framework.sce.evaluator;
+
+import dk.alexandra.fresco.framework.PerformanceLogger;
+import dk.alexandra.fresco.framework.PerformanceLogger.Flag;
+import dk.alexandra.fresco.framework.ProtocolCollection;
+import dk.alexandra.fresco.framework.builder.ProtocolBuilder;
+import dk.alexandra.fresco.framework.network.SCENetwork;
+import dk.alexandra.fresco.framework.network.SCENetworkSupplier;
+import dk.alexandra.fresco.framework.sce.resources.ResourcePool;
+import java.io.IOException;
+
+public class BatchedProtocolEvaluatorPerformanceDecorator<ResourcePoolT extends ResourcePool, Builder extends ProtocolBuilder>
+    extends BatchedSequentialEvaluator<ResourcePoolT, Builder> {
+
+  private PerformanceLogger pl;
+
+  public BatchedProtocolEvaluatorPerformanceDecorator(PerformanceLogger pl) {
```

Comment:
So just to be clear: you're saying make the evaluator decorator take an evaluator as a constructor parameter? If this is the case, it doesn't quite work since processBatch is called within the evaluator itself, thus avoiding any decorater. If this is not the case, please state what you intent (or drop by and explain :) )

Reply:
Currently there exists two different designs for evaluating batches. 
SequentialEvaluator has it's own internal
BatchedSequentialEvaluator uses the BatchStrategy
Introduce an interface that evaluates a batch of protocols and use it both places with a different implementation. Add a third which is decorating performance logging